src/components/MapManager.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState } from 'react';
+import { useSelector } from 'react-redux';
+
+import MapView from './MapView';
+import DashboardSidebar from './DashboardSidebar';
+import { selectAuthIsLoggedIn } from '../redux/selectors/authSelector';
+
+import '../stylesheets/mapManager.scss';
+
+const MapManager = () => {
+
+  const [propertyGrid, setPropertyGrid] = useState(null);
+
+  const isLoggedIn = useSelector(selectAuthIsLoggedIn);
+
+  function updatePropertyGrid(newGrid) {
```

Comment:
To be deleted when suggestion implemented. I know its my piece of code by the way  :P